acceptance/acceptance_test.go

```diff
@@ -27,13 +27,13 @@ func TestMain(m *testing.M) {
 	}))
 }
 
-func TestPullRequests(t *testing.T) {
```

Comment:
It's a shame about this diff because it's going to make it kinda janky to track back the git history. It's not the end of the world though. Maybe once we're happy with this PR we can squash it all down and only have the addition of `TestSecrets` in this part.

Reply:
Aside from being able to skim the list of tests, the conflict with these tests being added at the end of the ever growing list felt like it could benefit from being alphabetized 🤷

Reply:
Ah, from the diff it just looked like things had been bounced around by accident. Sure, is fine with me.

Reply:
For now, I'm just going to undo my reorganizing changes around the test functions.  I don't like it but want to get this done despite value in being able to scan the test functions alphabetically.